docs/installation/instructions.rst

```diff
@@ -1,10 +1,19 @@
 Installation Instructions
 =========================
 
+Supported Platforms
+-------------------
+
+Pulp should work on any operating system that can provide a Python 3.5+ runtime environment and
+the supporting dependencies e.g. a database, and has been demonstrated to work on Ubuntu, Debian,
```

Comment:
Can this become two sentences? The second sentence would start with "Pulp has been ...".